apps/taxes/src/modules/app/webhook-response.test.ts

```diff
@@ -0,0 +1,50 @@
+import { NextApiResponse } from "next";
+import { WebhookResponse } from "./webhook-response";
+import { describe, it, expect, vi } from "vitest";
+import { ExpectedError } from "../taxes/tax-provider-error";
+
+describe("WebhookResponse", () => {
+  it("returns 500 when thrown unexpected error", () => {
+    const jsonMock = vi.fn();
+    const statusMock = vi.fn().mockReturnValueOnce({ json: jsonMock });
+
+    const mockResponse = {
+      status: statusMock,
+    } as unknown as NextApiResponse;
+    const webhookResponse = new WebhookResponse(mockResponse);
+    const unexpectedError = new Error("Unexpected error");
+
+    webhookResponse.error(unexpectedError);
+
+    expect(statusMock).toHaveBeenCalledWith(500);
+  });
+  it("returns 200 when thrown expected error", () => {
+    const jsonMock = vi.fn();
+    const statusMock = vi.fn().mockReturnValueOnce({ json: jsonMock });
+
+    const mockResponse = {
+      status: statusMock,
+    } as unknown as NextApiResponse;
+    const webhookResponse = new WebhookResponse(mockResponse);
+    const expectedError = new ExpectedError("Expected error", { cause: "taxjar_no_nexus" });
```

Comment:
can you explain this? Why it returns 200 if there is an error?

Reply:
not used anymore